src/mako/index.html.mako

```diff
@@ -80,44 +68,77 @@ function onCopy(e) {
   e.clipboardData.setData('text/plain', installation_script);
   e.preventDefault()
 }
-</script>
-	<title>${title}</title>
+  </script>
+  <title>${title}</title>
 </head>
 <body>
-<H1>${title}</H1>
-<ul>
-% for an in sorted(api.list.keys()):
-    % if an in api.blacklist:
+  <div class="container">
+  <h1>${title}</h1>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>API Name</th>
+        <th>API Docs</th>
+        <th>CLI Docs</th>
+        <th>Install</th>
+      </tr>
+    </thead>
+    <tbody>
+    % for name in sorted(api.list.keys()):
+      % if name in api.blacklist:
         <% continue %>\
-    % endif
-    % for v in api.list[an]:
+      % endif
+    % for version in api.list[name]:
+      <tr>
         <% 
-            type_names = tc.keys()
-            with open(api_json_path(directories.api_base, an, v)) as fp:
-              api_data = json.load(fp)
+            # We know type_names is just ["api", "cli"]
+            #type_names = tc.keys()
+            type_names = ["api", "cli"]
```

Comment:
I don't want to be nitpicky, but what is the reason for not procedurally obtaining information that is available already, but to hardcode it? This will be a bug once there is another class of derivative, like `gui` maybe?